app/my_api/v2/auth.py

```diff
@@ -181,11 +181,56 @@ def post(self):
                 access_token = create_access_token(identity=email)
                 refresh_token = create_refresh_token(identity=email)
                 return {'Message': 'Logged in as {}'.format(email),
-                        'Access Token': access_token}, 201
+                        'Access Token': access_token}, 200
             else:
                 return {'Message': 'Invalid credentials'}, 403
 
         except (Exception, psycopg2.DatabaseError) as error:
             cur.execute("rollback;")
             print(error)
             return {'Message': 'current transaction is aborted'}, 500
+
+
+class ResetPassword(Resource):
+    """docstring for ResetPassword."""
+    parser = reqparse.RequestParser()
+    parser.add_argument(
+        'email',
+        type=str,
+        required=True,
+        help="Email field empty"
+    )
+    parser.add_argument(
+        'password',
+        type=str,
+        required=True,
+        help="Password field empty"
+    )
+    parser.add_argument(
+        'passwordconfirm',
+        type=str,
+        required=True,
+        help="Confirm password field empty"
+    )
+
+    @classmethod
+    def put(self):
+        data = ResetPassword.parser.parse_args()
+        email = data['email']
+        password = data['password']
+        passwordconfirm = data['passwordconfirm']
+        while True:
+            if not email:
+                return {'Message': 'Email cannot be empty'}, 400
+            elif not password:
```

Comment:
Consider a situation where I know the admin's email address, following this logic it seems it would be possible for me to change his password and take over his account and/or identity. What are your thoughts on this?

Reply:
Yah, I see that big picture. It's very possible to change their credentials. I will improve my logic